action.yml

```
@@ -155,7 +163,38 @@ runs:
       run: |
         [[ "${{ inputs.serverURL }}" ]] || { echo "Error: Missing server URL" ; exit 1; }
         [[ "${{ inputs.serverURL }}" = *[!\ ]*  ]] || { echo "Error: Server URL consists of empty spaces only." ; exit 1; }
-        COMMANDSTR="${{ inputs.serverURL }} ${{ inputs.testConfigID }} --client-id=${{ inputs.clientID }} --request-timeout=${{ inputs.requestTimeout }} --request-retries=${{ inputs.requestRetries }} --backoff-factor=${{ inputs.backoffFactor }} --poll-interval=${{ inputs.pollInterval }} --test-environment-timeout=${{ inputs.testEnvironmentTimeout }} --log-level=${{ inputs.logLevel }}"
+
+        echo ">> Checking test config info..."
+        
+        if [[ -n "${{ inputs.testConfigID }}" ]]; then
+          echo "ID=${{ inputs.testConfigID }}"
+          COMMANDSTR="${{ inputs.serverURL }} ${{ inputs.testConfigID }}"
+        elif [[ -n "${{ inputs.testConfigName }}" ]]; then
+          echo "Name=${{ inputs.testConfigName }}"
```

```suggestion
          echo "Test Config Name=${{ inputs.testConfigName }}"
```
or should we use testConfigName? Because the validation message is using the actual argument name.
either one is ok I guess. Just a suggestion